src/sdkV2.ts

```diff
@@ -490,4 +522,14 @@ export default class CegaEvmSDKV2 {
     const oracleEntry = await this.loadOracleEntry();
     return oracleEntry.getPrice(baseAsset, quoteAsset, timestamp, oracleDataSource);
   }
+
+  async pythUpdateAssetPrices(
+    timestamp: number,
+    assetAddresses: EvmAddress[],
+    updates: string,
+    fee: number,
```

Comment:
fee should be a `ethers.BigNumberish`

if you're submitting a value, it can be a bignumber